src/styles/composeBoxStyles.js

```diff
@@ -67,6 +67,7 @@ export default ({ color, backgroundColor, borderColor }: Props) => ({
   composeMenu: {
     flexDirection: 'row',
     overflow: 'hidden',
+    backgroundColor: 'transparent',
```

Comment:
The style is very weird (as I mentioned in the last PR review :) )
I'd add a comment here so we don't accidentally remove it again.

Reply:
@borisyankov Added comment :) 👍